lib/pages/chat/chat_input_row_web.dart

```diff
@@ -54,39 +52,33 @@ class ChatInputRowWeb extends StatelessWidget {
           Expanded(
             child: inputBar,
           ),
-          KeyBoardShortcuts(

```

Comment:
Why remove it?

Reply:
Because Shortcuts are now in `_buildInputBar` which is a child widget of `ChatInputRowWeb`. This allow to use shortcuts whenever the input bar is used. Also this widget did not work. My thought is that it used the wrong focus scope when put inside `chat_input_row_web.dart`

Reply:
Here the related lines:

https://github.com/linagora/twake-on-matrix/blob/91419cffde7bbf0a6661589edbce39f02b82bf1d/lib/pages/chat/chat_input_row.dart#L74-L156